src/schemathesis/hooks.py

```diff
@@ -222,6 +222,11 @@ def before_load_schema(context: HookContext, raw_schema: Dict[str, Any]) -> None
     """Called before schema instance is created."""
 
 
+@HookDispatcher.register_spec([HookScope.GLOBAL])
+def before_one(strategy: st.SearchStrategy) -> st.SearchStrategy:
+    """Creates an additional test and is called before one valid request."""
```

Comment:
The hook name sounds a bit confusing to me since it's behavior actually implies generating new tests. May be something `generate_tests`? or something like that, maybe we can reflect also that these tests are generated per endpoint

Reply:
I updated the name to `generate_test_per_operation`. This is verbose but descriptive, so please let me know if we should keep refining the name.